seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -168,11 +168,11 @@
       "id": "a56138aff60341a09ed6c480",
       "title": "Inventory Update",
       "description": [
-        "Compare and update inventory stored in a 2d array against a second 2d array of a fresh delivery. Update current inventory item quantity, and if an item cannot be found, add the new item and quantity into the inventory array in alphabetical order.",
+        "Compare and update the inventory stored in a 2D array against a second 2D array of a fresh delivery. Update the current existing inventory item quantities (in <code>arr1</code>). If an item cannot be found, add the new item and quantity into the inventory array. The returned inventory array should in alphabetical order by item.",
```

Comment:
The sentence at the end should be 'The returned inventory array should __be__ in alphabetical order by item.'